prepare-vms/scripts/trainer-cli

```diff
@@ -333,8 +331,8 @@ run_cli() {
         describe_tag $TAG
         tag_is_reachable $TAG
         echo "You may be interested in running one of the following commands:"
-        echo "trainer ips $TAG"
-        echo "trainer deploy $TAG <settings/somefile.yaml>"
+        echo "./trainer ips $TAG"
+        echo "./trainer deploy $TAG settings.yaml"
```

Comment:
Yes; I would like to progressively move away from the situation where we are using local symlinks (or copies) as state. It makes it harder to deal with multiple groups of machine simultaneously. Eventually I'd love to get rid of `ips.txt` too; or rather: use it as an output-only file, never read from it. (Which is almost the case but not quite! :))